src/mattack_actors.cpp

```diff
@@ -847,20 +875,54 @@ bool gun_actor::call( monster &z ) const
             }
             return false;
         }
-
+        aim_at = target->pos();
     } else {
         target = z.attack_target();
         if( !target || !z.sees( *target ) || ( !target->is_monster() && !z.aggro_character ) ) {
-            return false;
+            //return false;
+            if( !target_moving_vehicles ) {
+                return false;
+            }
+            //No living targets, try to find a moving car
+            untargeted = true;
+            vehicle *veh = find_target_vehicle( z, get_max_range() );
+            if( !veh ) {
+                return false;
+            }
+            std::vector<tripoint> valid_targets;
+            std::vector<tripoint> visible_points;
+            for( const tripoint &p : veh->get_points() ) {
+                if( !z.sees( p ) ) {
+                    continue;
+                }
+                visible_points.push_back( p );
+                if( veh->part_with_feature( p, VPFLAG_CONTROLS, true ) >= 0 &&
+                    veh->part_with_feature( p, VPFLAG_ENGINE, true ) >= 0 &&
+                    veh->part_with_feature( p, VPFLAG_WHEEL, true ) >= 0 ) {
```

Comment:
These want an index, not a point. Either an overload with a point needs to be added or this condition needs to iterate through the parts at this point and see if any of them have these flags.